src/hw-2/index.ts

```diff
@@ -0,0 +1,75 @@
+import {getAutoSuggestUsers, getUserIndex} from './utils';
+import {User} from './types';
+import {validation} from './validation';
+import * as core from 'express-serve-static-core';
+
+const express = require('express');
+const app = express();
+const router: core.Router = express.Router();
+
+const PORT = 3000;
+const {v1: uuidv4} = require('uuid');
+const userStore: User[] = [];
+
+app.listen(PORT, () => {
+    console.log(`Server is started on port ${PORT}`)
+});
+
+app.use(express.json());
+
+router.route('/user/:id')
+    .get((req: any, res: any, next: any) => {
+        const user: User = userStore.find((item: User) => item.id === req.params.id);
+
+        Boolean(user && !user.isDeleted)
```

Comment:
if you'd like you can use 'Boolean(!user?.isDeleted)'

Reply:
Fixed. Thanks.